adafruit_ble/characteristics/json.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: 2022 Mark Raleson
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`json`
+====================================================
+
+This module provides Json characteristic.
+
+"""
+
+import json
+from . import Attribute
+from . import Characteristic
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_BLE.git"
+
+
+class JsonCharacteristic(Characteristic):
```

Comment:
If you make this a subclass of `StringCharacteristic`, that would seem to save a few encoding/decoding steps?

Reply:
I did this initially, but because of the packing/unpacking of initial_value in init and also in the setter getter inheriting from StringCharacteristic ended up not really saving any typing and added an additional inheritance layer that hid attributes like max_length.